.gitignore

```diff
@@ -34,3 +35,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# data
+/data
```

Comment:
We can't exclude this because it will prevent data being auto-updated on the other branch. The script writes into this directory